projects/manageiq.py

```diff
@@ -77,6 +77,12 @@ def deploy():
             'username: root', 'username: {0}'.format(env.user))
         run('bin/rake db:migrate')
 
+    # enables container management UI
+    with cd('manageiq/vmdb'):
+        sed('config/vmdb.tmpl.yml',
+            'maindb: ExtManagementSystem',
+            'maindb: ExtManagementSystem \\n  containers: true')
```

Comment:
@dkorn I think this introduces a white space after `ExtManagementSystem`. I think it should be:

    ...
    'maindb: ExtManagementSystem\\n  containers: true')